ru/src/README_RU.md

```diff
@@ -0,0 +1,104 @@
+# Real Time For the Masses
+
+Конкурентный фреймворк для создания систем реального времени.
+
+**ВАЖНО** Эта библиотека в настоящее время в состоянии пре-релиза (бета).
```

Comment:
This note has been removed from the english version. Could you remove it here as well?